proto/auth.proto

```diff
@@ -69,8 +69,46 @@ message _GenerateApiTokenRequest {
         Expires expires = 2;
     }
 
-    // session token recieved from `momento login` command
-    string session_token = 3;
+    // Aliases for categories of functionality.
+    enum CacheRole {
+      CachePermitNone = 0;
+      // Restricts access to apis that read and write data from caches: No higher level resource description or modification.
+      CacheReadWrite = 1;
+    }
+
+    // Aliases for categories of functionality.
+    enum TopicRole {
+      TopicPermitNone = 0;
+      // Restricts access to apis that read and write data from topics: No higher level resource description or modification.
+      TopicReadWrite = 1;
+    }
+
+    string auth_token = 3;
```

Comment:
This is totally a nitpick... could you move this up above the enums? Otherwise it's just sort of stranded here having nothing to do with permissions 😆 

Separately... how are we going to role out this name change on the server? I think since the console is already using this api we maybe need to deprecate `session_token` and add this as a new value?

Reply:
as long as the type and the index are the same, the name doesn't matter... protobuf just packs the values as binary data, names aren't used in the serialization

Reply:
oh, ok, cool!